### PR TITLE
chore: update to the new bundle publish flow

### PR DESCRIPTION
**Pull request by Racer159**
## Description

This updates gitlab to use the bundle publish flow added in: https://github.com/defenseunicorns/uds-common/pull/73

## Related Issue

Relates to https://github.com/defenseunicorns/uds-software-factory/issues/18

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Other (security config, docs update, etc)

## Checklist before merging

- [X] Test, docs, adr added or updated as needed
- [X] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-package-gitlab/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

**Comment by Racer159** [comment]
(draft since this will need a uds-common release)

File: tasks/publish.yaml [hunk 1]
```diff
@@ -31,3 +19,37 @@ tasks:
...
+        task: create:test-bundle
+        with:
+          options: -a amd64
+
+      - description: Create the ARM64 test bundle
+        task: create:test-bundle
+        with:
+          options: -a arm64
+
+      - description: Publish the packages
```
Comment (zachariahmiller) [inline tasks/publish.yaml:50]: publish the bundle?

**Comment by Racer159** [comment]
tested: https://hub.docker.com/r/racer159/gitlab-test/tags
image

File: .github/workflows/tag-and-release.yaml [hunk 2]
```diff
@@ -49,7 +49,9 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Publish Package
-        run: uds run -f tasks/publish.yaml package  --set FLAVOR=${{ matrix.flavor }}
+        run: |
+          uds run -f tasks/publish.yaml package --set FLAVOR=${{ matrix.flavor }}
+          uds run -f tasks/publish.yaml test-bundle --set FLAVOR=${{ matrix.flavor }}
```
Comment (zachariahmiller) [inline .github/workflows/tag-and-release.yaml:55]: cant select the correct lines in this view but did you want to bump the other actions to 0.3.2 as well?

File: tasks.yaml [hunk 3]
```diff
@@ -2,11 +2,11 @@ includes:
   - cleanup: ./tasks/cleanup.yaml
   - dependencies: ./tasks/dependencies.yaml
   - test: ./tasks/test.yaml
-  - create: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.1/tasks/create.yaml
-  - lint: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.1/tasks/lint.yaml
-  - pull: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.1/tasks/pull.yaml
-  - deploy: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.1/tasks/deploy.yaml
-  - setup: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.1/tasks/setup.yaml
+  - create: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.2/tasks/create.yaml
```
Comment (Racer159) [inline tasks.yaml:5]: this @zachariahmiller ?

File: .github/workflows/tag-and-release.yaml (hunk 2 above)
Comment (zachariahmiller) [reply .github/workflows/tag-and-release.yaml:55]: thanks

**Review by zachariahmiller — APPROVED** [review]
:shipit:

**Pull request merged**